New package: CUDSS v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: CUDSS
- Repository: https://github.com/exanauts/CUDSS.jl
- Created by: @amontoison
- Version: v0.1.0
- Commit: d0adcb7cf398673894c769e49669775b8f39b6ce
- Reviewed by: @amontoison
- Reference: https://github.com/exanauts/CUDSS.jl/commit/d0adcb7cf398673894c769e49669775b8f39b6ce#commitcomment-134665579
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6baaecec6b393544966617c99988c0695d6edd622cc25ca27b7600d8fcbfc8d26e6da807d8b80e6c9dfd7e563fa772c15c33a2b721559b6b1c356d3b82b85b8a3915c8965860c1a83f1f9c4a6211f05eb879a6b3756eeff546a28a4fcbde0c0809b6f2a02d71bb73a038e304fbfb8e1cd542e3ebc73c14fb9c3fb065a94d5d9b5b07cabd8185b643b79d21520bd5e8c8fa908971b4f6dab222650843e7e8911cc566704617ca2b996b4387f1f8d3261d3 -->

amontoison commented:
`CUDSS.jl` is a Julia interface to the new toolbox developed by NVIDIA for direct solvers of sparse linear systems. It is not included in `CUDA.jl` as the library is still in preview. I would like to keep the name `CUDSS` because it aligns with the naming convention we use for other CUDA libraries, such as `CUBLAS`, `CUSOLVER`, `CUSPARSE`, etc.

goerz commented:
Can you add that explanation to the README and documentation? Right now these do not make it clear that what the purpose of the package is.

amontoison commented:
Thanks for your comment @goerz! I updated the README.md and the documentation.

JuliaRegistrator committed:
New package: CUDSS v0.1.0

UUID: 45b445bb-4962-46a0-9369-b4df9d0f772e
Repo: https://github.com/exanauts/CUDSS.jl.git
Tree: c1a90b9ab3cf8cf0412fcaa46eb91a4f1a9606a3

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
